lib/assets/assets.go

```diff
@@ -82,22 +82,26 @@ func Serve(w http.ResponseWriter, r *http.Request, asset Asset) {
 func MimeTypeForFile(file string) string {
 	ext := filepath.Ext(file)
 	switch ext {
-	case ".htm", ".html":
```

Comment:
While we may not have any htm files it's not inconceivable that custom themes may have them, I think we should keep this.

Reply:
True, but then mime.TypeByExtension return the right result, including the utf-8 thing...

Reply:
It might, or it might not. The reason we have that thing is that the TypeByExtension thing depends on OS specific stuff  which is just not always there.

Reply:
Oh, right. Put it back in.